charts/kcp/templates/kcp.yaml

```diff
@@ -229,9 +250,32 @@ spec:
       labels:
         app: kcp
     spec:
+      {{- if .Values.kcp.hostAliases.enabled }}
+      hostAliases:
+        {{- toYaml .Values.kcp.hostAliases.values | nindent 6 }}
+      {{- end }}
+      {{- if .Values.kcp.storagePermissionsInitContainer }}
+      # https://docs.digitalocean.com/products/kubernetes/how-to/add-volumes/
+      initContainers:
+      - name: kcp-data-permission-fix
+        image: busybox
+        {{- if .Values.audit.enabled }}
+        command: ["/bin/chmod","-R","777", "/kcp /audit"]
+        {{- else }}
+        command: ["/bin/chmod","-R","777", "/kcp"]
+        {{- end }}
+        volumeMounts:
+        - name: kubeconfig
+          mountPath: /kcp
+        {{- if .Values.audit.enabled }}
+        - name: audit-log
+          mountPath: /audit
+        {{- end }}
+      {{- end }}
```

Comment:
I do not believe this is required. Have you tried setting `securityContext.fsGroup=65532`? It did not work for me without that on AWS either.

Reply:
We definitely should add the `fsGroup` security context though, so proper CSI drivers can do it.

Reply:
Fixed